payment-manager/src/main/java/com/qpay/paymentmanager/event/PaymentNotificationProducer.java

```diff
@@ -10,16 +10,16 @@
 @Slf4j
 @Component
 @RequiredArgsConstructor
-public class PaymentProducer {
+public class PaymentNotificationProducer {
 
     private final KafkaTemplate<String, PaymentNotification> kafkaTemplate;
 
-    @Value("${kafka.topics.payment-topic}")
+    @Value("${broker.topics.payment-topic}")
     private String paymentTopic;
 
-    public void sendPayment(final PaymentNotification paymentNotification) {
+    public void publishPaymentNotification(final PaymentNotification paymentNotification) {
         kafkaTemplate.send(paymentTopic, paymentNotification.emailTo(), paymentNotification);
-        log.info("Message send {}", paymentNotification);
+        log.info("Event of sending message to kafka: {}", paymentNotification);
```

Comment:
Strange log from english point of view. Maybe just `Payment notification event sent {}`